src/js/webrequest.js

```diff
@@ -465,16 +448,15 @@ function recordFingerprinting(tabId, msg) {
 /**
  * Read the frame data from memory
  *
- * @param tabId TabId to check for
- * @param frameId FrameID to check for
+ * @param tab_id Tab ID to check for
```

Comment:
To be clear, I like Python style for non-objects and Java style for objects but I'm willing to simplify.

Once we have more consistent rules, we could update our ESLint config to enforce style stuff, if we wish.